charts/sonarqube-lts/Chart.yaml

```diff
@@ -18,9 +18,11 @@ maintainers:
     email: tobias.trabelsi+helm@sonarsource.com
 annotations:
   artifacthub.io/changes: |
+    - kind: changed
+      description: "updated SonarQube to 8.9.7"
     - kind: fixed
```

Comment:
The previous fix can be removed from the list. It is already live on [artifacthub](https://artifacthub.io/packages/helm/sonarqube/sonarqube-lts?modal=changelog)